triggers/triggers.go

```diff
@@ -52,8 +53,27 @@ func GetTriggers(templatesCfg []NotificationTemplate, triggersCfg []Notification
 	return parseTriggers(triggersCfg, templates)
 }
 
+func spawnExprEnvs(opts map[string]interface{}) interface{} {
+	envs := map[string]interface{}{
+		"toTime": func(timestamp string) time.Time {
```

Comment:
We might want to provide more functions in the future. 

I think it would be good to provide time-related function in a time namespace/package. E.g. `time.now`. Can you please make this change?

Could you moving time-related functions into `time` 'namespace' so the user can access them using e.g. `time.now()` ?

Reply:
What a great idea. I will update this MR in a few hours.